src/pages/organizations/[organizationid].js

```diff
@@ -419,40 +419,48 @@ export default function Organization(props) {
           }}
         >
           <Divider varian="fullwidth" />
-          <Typography
-            sx={{
-              mt: [80 / 8, 80 / 4],
-            }}
-            variant="h4"
-          >
-            About the Organization
-          </Typography>
-          <Typography variant="body2" mt={7}>
-            <Box
-              component="span"
-              dangerouslySetInnerHTML={{
-                __html: marked.parse(organization.about || 'NO DATA YET'),
-              }}
-            />
-          </Typography>
-          <Typography variant="h4" sx={{ mt: { xs: 10, md: 16 } }}>
-            Mission
-          </Typography>
-          <Typography variant="body2" mt={7}>
-            <Box
-              component="span"
+          <article>
+            <Typography
```

Comment:
@RubenSmn I think we can use this:
```
<Typography component="article" />
```
It's clear

Reply:
The article here is only providing a landmark here. So I don't think we need the MUI `Typhography` component. However I just found a mistake that I overlooked when merging. I made [this pr to fix it](https://github.com/Greenstand/treetracker-web-map-client/pull/1506)